invoice/src/main/java/org/killbill/billing/invoice/dao/CBADao.java

```diff
@@ -132,7 +132,7 @@ public Set<UUID> doCBAComplexityFromTransaction(final Set<UUID> invoiceIds,
         final List<InvoiceModelDao> invoices = new ArrayList<>();
         for (UUID id : invoiceIds) {
             final InvoiceModelDao invoice = transInvoiceDao.getById(id.toString(), context);
-            invoiceDaoHelper.populateChildren(invoice, invoicesTags, entitySqlDaoWrapperFactory, context);
+            invoiceDaoHelper.populateChildren(invoice, invoicesTags, false, entitySqlDaoWrapperFactory, context); //TODO_1951 - should this be true?
```

Comment:
I think it should be ok with `false` - you can remove TODO: Since `isRepaired` is only needed when doing void operations, and those are not called during void, should be fine.